flink-state-backends/pom.xml

```diff
@@ -39,5 +39,6 @@ under the License.
 	<modules>
 		<module>flink-statebackend-rocksdb</module>
 		<module>flink-statebackend-heap-spillable</module>
+		<module>flink-statebackend-changelog</module>
```

Comment:
I think the interfaces definitions should be in the `flink-runtime`, probably along the in memory implementation. We know there are other usecases where we might want to use it, furthermore there will be different implementations of the changelog. 

Maybe file based implementation could be placed anywhere for the time being, but I would need to think about this structure a bit more.

Reply:
I'd say there **might** be use case where we could use. They are even close to design.
And it and **might** have different implementations.

I have concerns regarding both, but I don't mind putting the API part into the `flink-runtime`.